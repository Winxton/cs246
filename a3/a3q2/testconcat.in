r a hello
r b world
s c a b
wc
lc
ec
t c a 123
wc
lc
ea
eb
ec
q
