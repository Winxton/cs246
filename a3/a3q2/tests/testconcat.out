helloworld
10
hello123
8
